examples/wxWebMapApp/PolygonReader.h

```diff
@@ -14,7 +14,7 @@ class PolygonReader {
      * @param polygons Read polygons
      * @param metaData Name of each polygon
     */
-    PolygonReader(wxString const& filename, std::vector<std::vector<wxMapPoint>>& polygons, std::vector<wxString>& metaData);
+    PolygonReader(wxString const& filename, std::vector<std::vector<wxMapPoint>>& polygons, std::vector<wxString>& metaData, bool isPolygon = true);
```

Comment:
What does it mean if it not a polygon? Add a comment to this function to make it clear